### ISSUE TITLE
Upgrade to new Azure SDK Libraries

### ISSUE THREAD

**Issue by jongio** [Azure/Azure-Sentinel#1063]
Azure SDK has GA'd new libraries that should be used in this project for consistency and performance improvements.  Please see all library links here: https://aka.ms/azsdk/

azsdke2e

**Comment by preetikr** [comment]
Hi @jongio This repository has just different types of content like queries, workbooks, etc. that can be consumed in Azure Sentinel and some tools that can help integration with Azure Sentinel. Can you please provide context on where you see these new SDK libraries can be used? Thanks.

**Comment by jongio** [comment]
Definitely!  The new libs all start with "Azure." and support all the latest and greatest feature of the Azure service.  For example, this file can be upgrade to the new libs; https://github.com/Azure/Azure-Sentinel/blob/50c3b83af78808cdd94dff21732fc937d7a54a51/DataConnectors/AzureStorage/GetAzureStorageLogsFunction.cs

There are perf improvements in the new libs too.

**Comment by jongio** [comment]
@tzanl - Can you take this one?

**Comment by v-xuto** [comment]
@jongio  PR(https://github.com/Azure/Azure-Sentinel/pull/1185) fixed this issue, can we close this issue?
